packages/datagateway-common/src/card/cardView.component.tsx

```diff
@@ -95,7 +103,11 @@ export interface CardViewProps {
   onPageChange: (page: number) => void;
   onFilter: (filter: string, data: Filter | null) => void;
   onResultsChange: (page: number) => void;
-  onSort: (sort: string, order: Order | null) => void;
+  onSort: (
+    sort: string,
+    order: Order | null,
+    defaultSort: UpdateMethod
```

Comment:
this variable should be renamed to "updateMethod" or something because it currently doesn't make sense.